pkg/dao/schema/resource.go

```diff
@@ -64,6 +64,12 @@ func (Resource) Fields() []ent.Field {
 		property.ValuesField("attributes").
 			Comment("Attributes to configure the template.").
 			Optional(),
+		field.String("comment").
+			Comment("Comment of the revision.").
```

Comment:
Why do we store revision comment in resources?

Reply:
Resources pending on relationships are not applied immediately when API called. Using this comment when they are applied in scheduled tasks.

Reply:
What about `"change_comment": "Change comment of the resource."`

Reply:
It may also apply to revison to make it consistent and clear.

Reply:
updated